server/streamreader/librespot_stream.cpp

```diff
@@ -47,6 +47,7 @@ LibrespotStream::LibrespotStream(PcmListener* pcmListener, boost::asio::io_conte
     bool normalize = (uri_.getQuery("normalize", "false") == "true");
     bool autoplay = (uri_.getQuery("autoplay", "false") == "true");
     killall_ = (uri_.getQuery("killall", "true") == "true");
+    string extra_params = uri_.getQuery("params", "");
```

Comment:
The ProcessStream class already reads "params" into params_: 
```c++
ProcessStream::ProcessStream(PcmListener* pcmListener, boost::asio::io_context& ioc, const StreamUri& uri) : PosixStream(pcmListener, ioc, uri)
{
    params_ = uri_.getQuery("params");
...
```

So it should be enough to just replace
```c++
    params_ = "--name \"" + devicename + "\"";
```
with 
```c++
    params_ += "--name \"" + devicename + "\"";
```

Reply:
Though this would affect all other subclasses. Is this something you want?

Reply:
Hm, and the develop branch has more options than just "name" (namely, "wd_timeout"). If you don't mind I'll keep the current approach, WDYT?

Reply:
Now "params" is read twice: 
1. in ProcessStream: `params_ = uri_.getQuery("params");`
2. in LibrespotStream: `extra_params = uri_.getQuery("params", "");`

If you just replace in LibrespotStream:
```c++
 params_ = "--name \"" + devicename + "\"";
```
with 
```c++
 params_ += "--name \"" + devicename + "\"";
```
then params_ will contain the "extra params"

Reply:
Done, please take another look.